frontend/components/media/MediaMap.vue

```diff
@@ -6,10 +8,15 @@
 </template>
 
 <script setup lang="ts">
+
+import { onMounted } from 'vue';
+import { useI18n } from "vue-i18n";
+import maplibregl from 'maplibre-gl';
+import direction-arrow from "frontend/public/icons/direction-arrow.png";
```

Comment:
This is breaking the header rules here, @rabelmervin. I'll see what I can do to fix it 🤔😊